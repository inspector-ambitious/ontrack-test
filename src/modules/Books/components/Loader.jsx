import React from "react";
import { Spinner } from "react-bootstrap";
import "./Loader.scss";

const Loader = () => {
  return (
    <div className="booksLoaderContainer">
      <Spinner animation="border" />
    </div>
  );
};

export default Loader;
