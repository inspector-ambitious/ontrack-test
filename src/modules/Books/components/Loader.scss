.booksLoaderContainer {
  position: fixed;
  display: flex;
  justify-content: center;
  align-items: center;
  height: inherit;
  background: rgba(255, 255, 255, 0.5);
  z-index: 1;
  width: 100vw;
  height: 100vh;
}
