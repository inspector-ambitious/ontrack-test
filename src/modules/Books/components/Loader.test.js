import React from "react";
import { shallow } from "enzyme";
import Loader from "./Loader";

describe("Books/components/Loader", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
