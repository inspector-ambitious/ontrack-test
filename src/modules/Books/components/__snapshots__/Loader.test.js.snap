// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books/components/Loader should render without crashing 1`] = `
<div
  className="booksLoaderContainer"
>
  <Spinner
    animation="border"
  />
</div>
`;
